build_tools/setup_helpers/extension.py

```diff
@@ -64,6 +64,7 @@ def _get_srcs():
 def _get_include_dirs():
     dirs = [
         str(_ROOT_DIR),
+        str(_TP_BASE_DIR / 'transducer' / 'submodule' / 'include'),
```

Comment:
I think installing the resulting library and header in the same directory as sox, i.e. `<_TP_BASE_DIR>/install/ [lib | include]` would make things simpler.